.idea/codeStyles/Project.xml

```diff
@@ -238,20 +224,7 @@
       </arrangement>
     </codeStyleSettings>
     <codeStyleSettings language="kotlin">
-      <option name="KEEP_BLANK_LINES_IN_DECLARATIONS" value="1" />
-      <option name="KEEP_BLANK_LINES_IN_CODE" value="1" />
-      <option name="ALIGN_MULTILINE_PARAMETERS" value="false" />
-      <option name="CALL_PARAMETERS_WRAP" value="5" />
-      <option name="CALL_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
-      <option name="CALL_PARAMETERS_RPAREN_ON_NEXT_LINE" value="true" />
-      <option name="METHOD_PARAMETERS_WRAP" value="5" />
-      <option name="METHOD_PARAMETERS_LPAREN_ON_NEXT_LINE" value="true" />
-      <option name="METHOD_PARAMETERS_RPAREN_ON_NEXT_LINE" value="true" />
-      <option name="EXTENDS_LIST_WRAP" value="5" />
-      <option name="METHOD_CALL_CHAIN_WRAP" value="5" />
-      <option name="METHOD_ANNOTATION_WRAP" value="1" />
-      <option name="FIELD_ANNOTATION_WRAP" value="1" />
-      <option name="ENUM_CONSTANTS_WRAP" value="5" />
+      <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
```

Comment:
Do we have to specify `CODE_STYLE_DEFAULTS` as `KOTLIN_OFFICIAL`? If we don't define `CODE_STYLE_DEFAULTS`, isn't the default value `KOTLIN_OFFICIAL` already?

Reply:
That was my first thought as well, but I found that if we don't have the CODE_STYLE_DEFAULTS set explicitly to `KOTLIN_OFFICIAL` then AS uses its own defaults (which seem to match "Kotlin obsolete IntelliJ IDEA codestyle" from the CodeStyle settings).

![image](https://user-images.githubusercontent.com/5091503/210785871-b4d636f8-03aa-48a1-a4db-335b491b285f.png)

See this video I made checking the `Project.xml` on a new project and changing codestyle settings. _Note: for better immersion while watching the video, imagine myself shrugging in disbelief at the end of it!_

https://user-images.githubusercontent.com/5091503/210786489-92d56cd5-3f54-45e4-954f-ce898cb3d8a3.mov